tls/extensions/s2n_server_alpn.h

```diff
@@ -18,3 +18,5 @@
 #include "tls/extensions/s2n_extension_type.h"
 
 extern const s2n_extension_type s2n_server_alpn_extension;
+
+bool s2n_alpn_should_send(struct s2n_connection *conn);
```

Comment:
You've got a separate "s2n_client_alpn_should_send", so shouldn't this be "s2n_server_alpn_should_send"?